docs/api/categories.md

```diff
@@ -0,0 +1,265 @@
+# Cagegories
+
+Categorize Speech Challenges or categories.
+
+## List all top level categories
+
+List all top level categories which do not have a parent category.
+
+### URL
+
+```http
+GET /categories HTTP/1.1
+```
+
+### Request
+
+```http
+GET /categories HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+The resonse is a JSON list with categories.
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+  {
+    "id": "category_1",
+    "created": "2017-01-12T09:35:20Z",
+    "updated": "2017-01-12T09:35:20Z",
+    "name": "Category 1",
+    "description": "Some awesome description.",
+    "color": "#00f",
+    "imageUrl": "https://api.itslanguage.nl/download/UKbsMpBsXaJUsBbK",
+    "iconUrl": "https://api.itslanguage.nl/download/GdExSbs-ZVNnQUUe",
+    "categories": ["category_1_1", "category_1_2"],
```

Comment:
Not necessarily. It could be something like https://api.itslanguage.nl/categories?parent_category=categorie_1. That requires no extra queries. However, the developer could also think of that himself...